### ISSUE TITLE
Issue with creating new companions

### ISSUE THREAD

**Issue by JasonMKY** [SanathTech/ai-companion#1]
Hello, I discovered it when I was browsing CodeWithAntonio's discord.

I know you may still be working on this project but, I have one issue, when clicking create button on the sidebar it redirects to "edit your companion" page instead of "/companion/new".

Somehow, when I enter this "/companion/new" in the URL bar it just loads the homepage and creating new companion is inaccessible. 
How can this problem be resolved?

**Comment by SanathTech** [comment]
Hi Jason,

Thanks for your email - but it's actually not a bug, I set up the functionality that way so that users can't create more than one companion each and spam the app.

You'll notice that it will allow you to create the first companion using the create button, but the next time you click it you'll only have the option to edit it.

Thanks,
Sanath


________________________________
From: JasonMKY ***@***.***>
Sent: Thursday, October 5, 2023 1:27:53 pm
To: SanathTech/ai-companion ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [SanathTech/ai-companion] Issue with creating new companions (Issue #1)


Hello, I discovered it when I was browsing CodeWithAntonio's discord.

I know you may still be working on this project but, I have one issue, when clicking create button on the sidebar it redirects to "edit your companion" page instead of "/companion/new".

Somehow, when I enter this "/companion/new" in the URL bar it just loads the homepage and creating new companion is inaccessible.
How can this problem be resolved?

**Comment by JasonMKY** [comment]
Oh true, didn't think of that. But is there anyway to create/bypass it?

Could you add a timer possibly?

**Comment by SanathTech** [comment]
Nope, only option I've allowed is to edit your current companion and change all the details, which will allow you to create a new companion.

________________________________

**Comment by JasonMKY** [comment]
Okay, what files can I modify to make edits to this feature?
